services/worker/src/worker/job_runners/parquet_and_dataset_info.py

```diff
@@ -774,12 +795,17 @@ def compute_parquet_and_dataset_info_response(
     parquet_files: List[ParquetFile] = []
     dataset_info: dict[str, Any] = {}
     for config in config_names:
+        ds_config_info = get_dataset_config_info(
```

Comment:
`get_dataset_config_info` calls `load_dataset_builder` (that we also call below). Also: it can rely on streaming and fail if the dataset does not support streaming, right? In that case, it defeats the purpose of downloading the dataset. Maybe handle the case with a try/except block?